spec/decorators/usps_decorator_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+class DummyController < ApplicationController
+  def initialize
```

Comment:
Yeah, instead of a real controller, just mock one:

```ruby
controller = instance_double('ApplicationController', user_session: session)
```

in the tests